python/py-htseq/Portfile

```diff
@@ -35,9 +35,8 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-pysam \
                     port:py${python.version}-numpy
 
-    notes-append \
-"If you want to produce plots or use htseq-qa-${python.branch},
-you will also need to install py${python.version}-matplotlib."
+    depends_run-append \
+                    port:py${python.version}-matplotlib
 
     livecheck.type  none
 } else {
```

Comment:
this `else` block is unneeded, as it is set as the default by the python PG

Reply:
Removed.